cbv3_django_prototype/config/settings/base.py

```diff
@@ -42,9 +42,21 @@
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
 
+#DATABASES = {
+#    "default": env.db("DATABASE_URL", default="postgres:///cbv3_django_prototype")
+#}
+
 DATABASES = {
-    "default": env.db("DATABASE_URL", default="postgres:///cbv3_django_prototype")
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'cbv3_django_prototype',
+        'USER': 'postgres',
```

Comment:
Lets use environment variables for database credentials and path.

Reply:
Oh, certainly.  This was done to avoid confusion because there weren't any instructions and many new users don't know how to deal with ENV Vars.  Would you mind putting some docs together on how you set/use Env vars for this?  I'll go ahead and change the settings file in the next iteration to use ENV Vars and make an issue for documentation.  Thanks!